Plugin.pm

```diff
@@ -628,6 +635,164 @@ sub getFeatured {
 	});
 }
 
+sub getHome {
+	my ( $client, $cb, $args, $params ) = @_;
+
+	getAPIHandler($client)->page(sub {
+		my $modules = shift;
+
+		my $items = [ map {
+			{
+				name => $_->{title},
+				type => 'link',
+				url => $_->{type} eq 'HIGHLIGHT_MODULE' ? \&getHighlights : \&getModule,
+				passthrough => [ { module => $_ } ],
+			}
+		} @$modules ];
+
+		$cb->( {
+			items => $items
+		} );
+	}, 'pages/home' );
```

Comment:
Please create a thin `P::T::API::Async->home()` wrapper. That path is an API specific thing which should not be spread across the code.

Reply:
I did not use them but you can it a lot of other pages with just one module. This is what I did initially and hence I've left that for the future. In fact, in every module, you have a 'showMore' key that contains a "apiPath" key like that
```
"showMore": {
                        "title": "View all",
                        "apiPath": "pages/single-module-page/2ebb02d0-017f-43de-889f-a03fef7216b3/100/992aa07f-21fb-49cd-89c4-17742dcb83b1/2"
                    },
```
That you can use to access the module directly. So I think that "page" call should be generic and the path is a parameter